Yoast/javascript
# Merge Norwegian feature branch

## Summary


This PR can be summarized in the following changelog entry:

* [yoastseo] Adds function words for Norwegian.
* [yoastseo] Adds Norwegian stemmer.
* [Yoast SEO Free] Improves keyphrase recognition in Norwegian by filtering out function words such as `tredje`, `deres`, `noen`.
* [Yoast SEO Premium] Introduces word form support for Norwegian.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Set your website to Norwegian
* In Yoast Seo Free:
     * Create a new post and add a text with at least 300 words (e.g. generated from https://no.lipsum.com/)
     * Set your keyphrase to `nitten katter`
     * Add the word `katter` to the text
     * Make sure that the keyphrase density shows that the keyphrase is found once
* In Yoast SEO Premium:
     * Create a new post and add text with at least 300 words
     * Set your keyphrase to `skikke`
     * Enter the form `skikka` the text/meta description/title. The relevant assessments should recognise
`skikka` as an instance of the keyphrase. You should see the following assessment feedback:
          * `Keyphrase in meta description: Keyphrase or synonym appear in the meta description. Well done!`
          * `Keyphrase density: The focus keyphrase was found 1 time. That's less than the recommended minimum of 3 times for a text of this length. Focus on your keyphrase!` (Number depending on how often you added the keyphrase in the text)
          * `Keyphrase in title: Does not contain the exact match. Try to write the exact match of your keyphrase in the SEO title and put it at the beginning of the title.`
     * Switch the keyphrase and the form in the text/title/meta description and make sure the results are still the same.






## Impact check

* This PR affects the following parts of the plugin, which may require extra testing:
  *

## UI changes

## Quality assurance


Fixes https://yoast.atlassian.net/browse/LINGO-608
